extensions/blocks/contact-form/components/jetpack-newsletter-integration-settings.js

```diff
@@ -44,6 +55,190 @@ const NoConsentBlockSettings = () => {
 	);
 };
 
+const pluginStateEnum = Object.freeze( {
+	ACTIVE: 1,
+	INSTALLED: 2,
+	NOT_INSTALLED: 3,
+} );
+
+const CreativeMailPluginErrorState = ( { error } ) => {
+	return (
+		<p>
+			<em style={ { color: 'red' } }>
+				{ jetpackCreateInterpolateElement(
+					__(
+						'The plugin failed to install. <b /> Please check the <a>plugin information</a> for detailed requirements.',
+						'jetpack'
+					),
+					{
+						a: (
+							<ExternalLink href="https://wordpress.org/plugins/creative-mail-by-constant-contact" />
+						),
+						b: (
+							<span>{error}</span>
+						)
+					}
+				) }
+			</em>
+		</p>
+	);
+};
+
+const CreativeMailPluginIsInstalling = ( { isActivating } ) => {
+	return (
+		<Button isSecondary icon={ <Icon style={ { animation: 'rotation 2s infinite linear' } } icon="update" /> } disabled>
+			{!isActivating && __( 'Installing…', 'jetpack' ) }
```

Comment:
Could you try to reformat the file to avoid these? Our pre-commit hook usually does that for you, but it seems that it hasn't happened here.

Reply:
done.